apisix/init.lua

```diff
@@ -724,6 +724,7 @@ local function healthcheck_passive(api_ctx)
                                            port or api_ctx.balancer_port,
                                            host,
                                            resp_status)
+                return
```

Comment:
We should use break here?

Reply:
In the second loop, I think break and return have the same effect

Reply:
we should use break here.

Reply:
Sorry, I can't get your point. What does "the second loop' mean?

Reply:
Loop of the 727 line, the function ends when the loop terminates, so think break and return have the same effect
![image](https://user-images.githubusercontent.com/25243232/203505029-970e7ea1-f392-4664-a83a-3632350d98d3.png)

Reply:
in L730, we should use `break`?